node-repository/src/main/java/com/yahoo/vespa/hosted/provision/persistence/NodeSerializer.java

```diff
@@ -125,6 +133,17 @@ private void toSlime(Node node, Cursor object) {
         node.status().firmwareVerifiedAt().ifPresent(instant -> object.setLong(firmwareCheckKey, instant.toEpochMilli()));
         node.reports().toSlime(object, reportsKey);
         node.modelId().ifPresent(modelId -> object.setString(modelIdKey, modelId));
+        node.networkPorts().ifPresent(ports -> toSlime(ports, object.setArray(networkPortsKey)));
+    }
+
+    private void toSlime(NetworkPorts networkPorts, Cursor array) {
```

Comment:
`SerializationTest` should be updated to test serialization of these new fields.